### PR TITLE
Add containerHost information

### PR DESCRIPTION
**Pull request by codings-dan**
### What changes are proposed in this pull request?
Add containerHost information on the worker page.

### CODE REVIEW

File: core/common/src/main/java/alluxio/util/webui/NodeInfo.java [hunk 1]
```diff
@@ -39,7 +39,12 @@
    * @param workerInfo the worker info
    */
   public NodeInfo(WorkerInfo workerInfo) {
-    mHost = workerInfo.getAddress().getHost();
+    if (!workerInfo.getAddress().getContainerHost().equals("")) {
+      mHost = workerInfo.getAddress().getHost() + "("
+              + workerInfo.getAddress().getContainerHost() + ")";
```
Comment (apc999) [inline core/common/src/main/java/alluxio/util/webui/NodeInfo.java:44]: string concat is expensive. Use `String.format`?

Comment (codings-dan) [reply core/common/src/main/java/alluxio/util/webui/NodeInfo.java:44]: good idea,thsnks

**Review by maobaolong — APPROVED** [review]
Thanks for the contribution! it LGTM.

**Review by apc999 — APPROVED** [review]

**Comment by codings-dan** [comment]
@apc999 I think there is no style problem with the code at least, is there a problem with the ci test?

**Comment by apc999** [comment]
> @apc999 I think there is no style problem with the code at least, is there a problem with the ci test?

I will  merge this PR after CI passed

**Comment by maobaolong** [comment]
image
The failed CI check seems not caused by this PR, @apc999 Would you please to help to re-run the CI checks?

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**